test/util/test_lock.py

```diff
@@ -102,6 +108,57 @@ async def test_set_usercode(lock_schlage_be469, mock_command, uuid4):
     assert len(ack_commands) == 1
 
 
+async def test_set_usercodes(lock_schlage_be469, mock_command, uuid4):
```

Comment:
Let's add type annotations for new tests.
```suggestion
async def test_set_usercodes(
    lock_schlage_be469: Node, mock_command: MockCommandProtocol, uuid4: str
) -> None:
```